cs-xml/src/main/java/io/cloudslang/content/xml/utils/XmlUtils.java

```diff
@@ -208,7 +219,7 @@ private static XPath createXpath() {
      * @return the InputStream representation of a file or string
      * @throws IOException Exception in case something goes wrong
      */
-    private static InputStream getStream(String xml, String filePath) throws IOException {
+    public static InputStream getStream(String xml, String filePath) throws IOException {
```

Comment:
I think this is just a helper method in the XmlUtils class. Maybe we can keep it private, if it is not used externally. If it is used externally, maybe we should move it in a separate class, since it is not XML related.